pkg/html/html.go

```diff
@@ -727,6 +731,188 @@ func testImpactingComponents(testImpactingBugs []bugsv1.Bug) string {
 	return s
 }
 
+func summaryJobsFailuresByBugzillaComponent(report, reportPrev sippyprocessingv1.TestReport, endDay int, release string) string {
+	failuresByBugzillaComponent := util.SummarizeJobsFailuresByBugzillaComponent(report)
+	failuresByBugzillaComponentPrev := util.SummarizeJobsFailuresByBugzillaComponent(reportPrev)
+
+	s := fmt.Sprintf(`
+	<table class="table">
+		<tr>
+			<th colspan=4 class="text-center"><a class="text-dark" title="Bugzilla components ranked by maximum fail percentage of any job." id="BZToWorstJobImpact" href="#BZToWorstJobImpact">Bugzilla Components to Worst Job Impact</a></th>
+		</tr>
+		<tr>
+			<th>Variant</th><th>Latest %d days</th><th/><th>Previous 7 days</th>
+		</tr>
+	`, endDay)
+
+	bzGroupTemplate := `
+		<tr class="%s">
+			<td>
+				%[2]s
+					<p>
+					<button class="btn btn-primary btn-sm py-0" style="font-size: 0.8em" type="button" data-toggle="collapse" data-target=".%[3]s" aria-expanded="false" aria-controls="%[3]s">Expand Failing Jobs</button>
+			</td>
+			<td>
+				%0.2f%% <span class="text-nowrap">(%d runs)</span>
+			</td>
+			<td>
+				%s
+			</td>
+			<td>
+				%0.2f%% <span class="text-nowrap">(%d runs)</span>
+			</td>
+		</tr>
+	`
+
+	naTemplate := `
+			<tr class="%s">
+				<td>
+					%[2]s
+					<p>
+					<button class="btn btn-primary btn-sm py-0" style="font-size: 0.8em" type="button" data-toggle="collapse" data-target=".%[3]s" aria-expanded="false" aria-controls="%[3]s">Expand Failing Jobs</button>
+				</td>
+				<td>
+					%0.2f%% <span class="text-nowrap">(%d runs)</span>
+				</td>
+				<td/>
+				<td>
+					NA
+				</td>
+			</tr>
+		`
+
+	bzFailingJobTemplate := `
+			<tr class="collapse %s">
+				<td>
+					%[2]s -- %[3]s
+					<p>
+					<button class="btn btn-primary btn-sm py-0" style="font-size: 0.8em" type="button" data-toggle="collapse" data-target=".%[4]s" aria-expanded="false" aria-controls="%[4]s">Expand Failing Tests</button>
+				</td>
+				<td>
+					%0.2f%% <span class="text-nowrap">(%d/%d failed/total)</span>
+				</td>
+				<td/>
+				<td>
+					NA
+				</td>
+			</tr>
+		`
+
+	for _, v := range failuresByBugzillaComponent {
+		safeBZJob := fmt.Sprintf("%s---component", v.Name)
+		safeBZJob = strings.ReplaceAll(safeBZJob, ".", "")
+		safeBZJob = strings.ReplaceAll(safeBZJob, " ", "")
+
+		prev := util.GetPrevBugzillaJobFailures(v.Name, failuresByBugzillaComponentPrev)
+		highestFailPercentage := v.JobsFailed[0].FailPercentage
+		lowestPassPercentage := 100 - highestFailPercentage
```

Comment:
so ultimately the value you're presenting in the table is "the percentage of jobs which did not fail because of this component" right?

i appreciate it is aligned w/ how the other tables work but it also doesn't seem like the most intuitive value.  Need to think about it.

Reply:
> i appreciate it is aligned w/ how the other tables work but it also doesn't seem like the most intuitive value. Need to think about it

aligning to the other tables was the what I was going for.  It was super confusing in the reverse and I knew what it was doing.

Reply:
hm, another confusing thing about this is that the top line value you're presenting isn't some aggregation of the percentage of failures this component is causing across all jobs.  It's "here's the job where this component performs worst"

which i realize probably aligns with your intent (if a single component is tanking a single job, that's the thing you care about more than anything in the world, more so than another component that might be impacting all jobs but to a lesser degree....which is more or less what the "test impacting components" view is showing).  But i'm pretty sure people won't understand it.  I know i was confused when i started expanding jobs and trying to understand how the top line number made sense in light of the expanded jobs values.

your hover text sort of explains it.  let me see if i can offer a suggestion on that to help.